lib/temperature_monitor/temperature_handler.ex

```diff
@@ -0,0 +1,91 @@
+defmodule TemperatureMonitor.TemperatureHandler do
+  use GenServer
+
+  def start_link(stash_pid) do
+    {:ok, _pid} = GenServer.start_link(__MODULE__, stash_pid, name: __MODULE__)
+  end
+
+  def init(stash_pid) do
+    filelist = TemperatureMonitor.Stash.get_value stash_pid
+    monitored_paths = []
+     {:ok, {filelist, stash_pid, monitored_paths}}
+  end
+
+  def callback(:stop) do
+    IO.puts "STOP"
+  end
+
+  def callback(filepath, events) do
+    IO.inspect {filepath, events}
+  end
+
+  def monitor_path(base_path) do
+    GenServer.cast(__MODULE__, {:add_base_path, base_path})
+  end
+
+  def get_monitored_paths do
```

Comment:
I couldn't figure out how to have the `Monitor` store the `base_path`.  You are right, in that it is fixed for life, but, I want to be able to pass it in from a config, rather than hard code it directly. 

What I wanted to do, was something like this:

```
def TemperatureMonitor.Monitor
  attr_accessible :base_path
end
```

I'm open to suggestions!